### ISSUE TITLE
Contracts: action is required before being able to turn in PVM contracts

### ISSUE THREAD

**Issue by itVictor** [redsunservers/LoadoutBugTracker#22]
Upon completing a Player v. Map contract, doing /turnin will not turn in the contract, nothing is displayed in chat and /progress will remain unchanged.

This can be fixed by reconnecting to the server, however I did notice doing /progress afterward showed that the counter had increased (eg it would show 2/1 after reconnecting).

Apparently this can be fixed without having to reconnect by doing /refreshcontract, however I forgot that command existed and had already turned it in before completing it.

in summary, action is always required on the player's part before they are able to turn in PvM contracts, as opposed to being able to turn it in as soon as progress is completed

**Comment by koopa516** [comment]
fixed
